<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ControlConfigDialog</class>
 <widget class="QDialog" name="ControlConfigDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>340</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>350</y>
     <width>171</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="controlConfigLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Control Configuration</string>
   </property>
  </widget>
  <widget class="Line" name="headerSeparator">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>301</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="footerSeparator">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>320</y>
     <width>301</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QGroupBox" name="colorGroupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>301</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="colorFormLabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Color:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="blueNameInput">
    <property name="geometry">
     <rect>
      <x>92</x>
      <y>80</y>
      <width>201</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QRadioButton" name="blueRadioButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>55</width>
      <height>26</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(52, 101, 164);</string>
    </property>
    <property name="text">
     <string>Blue</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="yellowRadioButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>73</width>
      <height>26</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(237, 212, 0);</string>
    </property>
    <property name="text">
     <string>Yellow</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="yellowNameInput">
    <property name="geometry">
     <rect>
      <x>92</x>
      <y>40</y>
      <width>201</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="teamGroupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>301</width>
     <height>80</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QWidget" name="formLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>40</y>
      <width>97</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="gkIdForm">
     <item row="0" column="0">
      <widget class="QLabel" name="gkIdLabel">
       <property name="text">
        <string>GK ID:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QSpinBox" name="gkIdSpinBox">
       <property name="maximum">
        <number>11</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="teamFormLabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Team:</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rightSideRadioButton">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>40</y>
      <width>61</width>
      <height>26</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Right</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="leftSideRadioButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>51</width>
      <height>26</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Left</string>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>leftSideRadioButton</tabstop>
  <tabstop>rightSideRadioButton</tabstop>
  <tabstop>gkIdSpinBox</tabstop>
  <tabstop>yellowRadioButton</tabstop>
  <tabstop>yellowNameInput</tabstop>
  <tabstop>blueRadioButton</tabstop>
  <tabstop>blueNameInput</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ControlConfigDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ControlConfigDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
